peercrawler_html_server.py

```diff
@@ -34,10 +35,11 @@ def bg_thread_func():
 
 
 def refresh_node_info():
-    global representative_mappings
+    global representative_mappings, mutex
 
     try:
-        representative_mappings = get("https://nano.community/data/representative-mappings.json").json()
+        with mutex:
```

Comment:
There is not much point locking it only when writing, it should also be locked when reading for the mutex to be effective.
There should be a getter function to lock and take a copy and return the copy.
I think the representative mapping could be a class of module on their own in their own file.